subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/TaskReportTask.java

```diff
@@ -245,7 +246,7 @@ private DefaultGroupTaskReportModel taskReportModelFor(Project project, boolean
         final SingleProjectTaskReportModel projectTaskModel = buildTaskReportModelFor(taskDetailsFactory, project);
         aggregateModel.add(projectTaskModel);
 
-        for (final Project subproject : project.getSubprojects()) {
+        for (final ProjectInternal subproject : ((ProjectInternal) project).getSubprojectsUnchecked()) {
```

Comment:
❌  Same as for `TaskDetailsFactory`, we seem to have changed the logic to ignore cross-project access violations instead of fixing them.

IIUC, this logic runs as part of the `TaskReportTask` task action. But with Isolated Projects, there is no guarantee that all necessary projects have been configured at this point (or have been configured at all).